e2e-tests/upgrade-consistency-sharded/conf/some-name.yml

```
@@ -10,6 +10,8 @@ spec:
   #platform: openshift
   image:
   imagePullPolicy: Always
+  # TODO: remove initImage after 1.18.0 release
+  initImage: perconalab/percona-server-mongodb-operator:main
```

maybe we need to do it via 
initImage: -init
```suggestion
  initImage: -init
```

```
--- a/e2e-tests/functions
+++ b/e2e-tests/functions
@@ -1040,6 +1040,7 @@ cat_config() {
        cat "$1" \
                | yq eval '(.spec | select(.image == null)).image = "'"$IMAGE_MONGOD"'"' \
                | yq eval '(.spec | select(has("pmm"))).pmm.image = "'"$IMAGE_PMM_CLIENT"'"' \
+               | yq eval '(.spec | select(has("initImage"))).initImage = "'"$IMAGE"'"' \
                | yq eval '(.spec | select(has("backup"))).backup.image = "'"$IMAGE_BACKUP"'"' \
                | yq eval '.spec.upgradeOptions.apply="Never"'
 }
```